test/jsmediatags-cross-tests.ts

```diff
@@ -108,13 +113,18 @@ const nodeTagsMissingValues = {
 describe('Cross tests jsmediatags', function() {
     it('write full', function() {
         jsmediatags.read(NodeID3.create(nodeTagsFull), {
-            onSuccess: (tag) => {
-                const tags = tag.tags
+            // @types/jsmediatags are kind of broken.
+            // E.g. tags.TXXX will return an array,
+            // but the types say it only returns an Object.
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            onSuccess: (tag: any) => {
```

Comment:
All the `any` casts here have a scope which is to wide, when casting we should reduce the scope as much as possible, otherwise this is the open door to too many issues.
It is better to embed the TXXX cast in a function, reuse it and document the issue in only on place.
Here we want to cast tags.TXXX with `as unknown as TagFrame[]` which is the correct expected type, this is the correct to fix a wrong type in TS.